pkg/flag/misconf_flags.go

```diff
@@ -53,6 +53,36 @@ var (
 		Value:      []string{},
 		Usage:      "Rego namespaces",
 	}
+	HelmValuesFileFlag = Flag{
+		Name:       "helm-values",
+		ConfigName: "misconfiguration.helm-values",
```

Comment:
Currently, the config format is like the following.

```
misconfiguration:
  helm-values:
    - values-prod.yaml
  helm-set:
    - key1=val1
  tf-vars:
    - foo.tfvars
    - bar.tfvars
```

Do you think grouping languages is too nested as below? I'm not sure which is better. I just would like to hear your thought.

```
misconfiguration:
  helm:
    values:
      - values-prod.yaml
    set:
      - key1=val1
  terraform:
    vars:
      - foo.tfvars
      - bar.tfvars
```

Reply:
Helm uses `--set` but I changed it to `--helm-set` to avoid collisions in the future with other configs that might be in the same files system that also wanted a `--set`

I like personally like how explicit the second config block is and I don't think it's too nested. @liamg - do you have an opinion?

Reply:
So I assume I would just change the `ConfigName` to `misconfiguration.helm.values` to achieve the second config block?

Reply:
>So I assume I would just change the ConfigName to misconfiguration.helm.values to achieve the second config block?

Yes. `misconfiguration.helm.values` should work.

Reply:
Changed to `misconfiguration.helm.xxxx` and `misconfiguration.terraform.xxx`